mas/mastest/src/androidTest/java/com/ca/mas/sample/testapp/tests/instrumentation/connecta/MASConnectaTests.java

```diff
@@ -591,6 +591,7 @@ public void onError(Throwable e) {
                 public void onError(Throwable e) {
                     result[1] = "Can't stop listening before starting.";
                     latch.countDown();
+                    latch.countDown();
```

Comment:
The CountDownLatch set to 2, just in case there is an error when startListeningToMyMessages, it didn't block the countdownlatch.await forever.